protocol-units/mempool/util/src/lib.rs

```diff
@@ -147,12 +147,12 @@ impl PartialOrd for MempoolTransaction {
 impl Ord for MempoolTransaction {
 	fn cmp(&self, other: &Self) -> Ordering {
 		// First, compare by slot_seconds
```

Comment:
This comment is outdated now.
```suggestion
		// First, compare by timestamps
```